src/FunctionApps/PITest_FunctionApp/IngestFunction/settings.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+
+class Settings:
+    hostname = os.environ.get("VDHSFTPHostname")
```

Comment:
where can we find documentation on where these are set? i presume in some function configuration on azure? are these variables initially created via terraform?